libs/ledger-live-common/src/families/cosmos/api/types.ts

```diff
@@ -264,14 +264,25 @@ export type GetRedelegations = {
       delegator_address: string;
       validator_src_address: string;
       validator_dst_address: string;
+      entries:
+        | {
+            creation_height: string;
+            completion_time: string;
+            initial_balance: string;
+            shares_dst: string;
+          }[]
+        // null since sdk 0.46
+        | null;
     };
     entries: {
-      creation_height: string;
-      completion_time: string;
-      initial_balance: string;
-      shares_dst: string;
-      unbonding_id: string;
-      unbonding_on_hold_ref_count: string;
+      redelegation_entry: {
+        creation_height: string;
+        completion_time: string;
+        initial_balance: string;
+        shares_dst: string;
+        unbonding_id: string;
```

Comment:
It seems optional as in your test `sdk v0.41/0.42/0.44/0.45` does contained such property.

Reply:
done